src/MPDevice_linux.cpp

```diff
@@ -52,6 +52,12 @@ MPDevice_linux::MPDevice_linux(QObject *parent, const MPPlatformDef &platformDef
         if (INVALID_VALUE == grabbed)
         {
             qWarning() << "Exclusive device grab wasn't successful: " << strerror(errno);
+            //Need a delay for setting MPDevice to WSServer
+            QTimer::singleShot(100,
+                [this]()
+                {
+                    sendNotification(tr("Exclusive Grab Failure"), tr("Moolticute couldn't open the USB port exclusively"));
```

Comment:
The translation will not work like that. The QTranslator is installed in the gui process, not in the daemon, and thus when the strings will be passed like that they will remain always in english

Reply:
Thanks I was not aware of this, so I moved the translation texts to GUI.